spec/loadbalancer/Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN ["/bin/bash", "-c", "wget -O ~/v1_index.sh https://raw.githubusercontent.com
 
 RUN chmod +x ~/v1_index.sh
 
-ENTRYPOINT ["~/v1_index.sh"]
+CMD ~/v1_index.sh
```

Comment:
there are some subtle way to use "entrypoint" rather than cmd. Say, what if I would want to pass some args? e.g. abc -123 -ok. Do you foresee such need for the LBer in the future? @ramaraosrikakulapu

Reply:
At the moment, I am not planning to pass arguments. But if we have to pass the arguments, we can execute the script 
 as - `~/v1_index.sh arg1 arg2`. Also, with ENTRYPOINT, pod is not running and throwing the error - 
```
Error: failed to start container "agentlber": Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"~/v1_index.sh\": stat ~/v1_index.sh: no such file or directory": unknown
```
@ayasuda-ge

Reply:
CMD may be replaced, and subject to be altered during the container runtime with command args ingestion. What are your test env and the args look like in your test?

Reply:
got it @ayasuda-ge . Will close the PR and try to work with ENTRYPOINT instead of CMD. Thank you !!